packages/types/src/type/SessionKeys.ts

```diff
@@ -13,22 +13,14 @@ import SessionKey from './SessionKey';
 export default class SessionKeys extends Struct {
   public constructor (value?: any) {
     super({
-      grandpaKey: SessionKey,
-      auraKey: SessionKey
+      ed25519: SessionKey
```

Comment:
Doesn't this break backward-compat?

Reply:
Yes and No. Keys were added in 2.x, so in this case it is ok to break the compat.